sysl2/sysl/seqs/diagutil.go

```diff
@@ -20,6 +20,10 @@ func OutputPlantuml(output, plantuml, umlInput string) {
 		plantuml = os.Getenv("SYSL_PLANTUML")
 	}
 
+	if plantuml == "" {
```

Comment:
Remove this and set default where you add the plantuml command line arg.

Reply:
yep, remove it.

Reply:
all the checks on `plantuml` should move to where the command line args are evaluated. Not here.
One place where it could be is to move lines 19-24 to seqs.go